CSC322-Grinnell/mayflower-directory
# Add rake task to seed the database with dummy data

Resolves #157 

# Changes

- Added a rake task (`rails db:dummy_seed`) to create fake profiles
- Fix profile image display code in development
- Move the logic for finding the profile image into the controller

# Testing

1. Run `rails test`
2. Run `rails db:dummy_seed`
3. Start the server
4. Go to `/pictures` and make sure a bunch of profiles show up for characters various plays
5. Click on the picture for "Caldwell Cladwell" and make sure his info shows up
